<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smart Calculator</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 245, 238);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F7F6F1;</string>
   </property>
   <widget class="QLineEdit" name="inputFunction">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>702</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 239, 213);
color: rgb(60, 60, 60);
font-size: 30px; </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>99</y>
      <width>700</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 239, 213);
color: rgb(60, 60, 60);
font-size: 30px; </string>
    </property>
    <property name="text">
     <string>RESULT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonX">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDot">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMinus">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(175, 175, 175);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}	</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMult">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(175, 175, 175);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}	</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(175, 175, 175);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}	</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDiv">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(175, 175, 175);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}	</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPow">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(175, 175, 175);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}	</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMod">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(175, 175, 175);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}	</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>280</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(175, 175, 175);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}	</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpen">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(175, 175, 175);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}	</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSin">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 127, 80);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCos">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 127, 80);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTan">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 127, 80);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAsin">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 127, 80);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAtan">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>440</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 127, 80);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAcos">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>440</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 127, 80);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAC">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>440</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBack">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPi">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>π</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonE">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>520</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(90, 90, 90);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>520</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 127, 80);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEqual">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>520</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font-size: 30px;
	background-color: rgb(255, 127, 80);
	border: 0px;
	border-radius: 12px;
}

QPushButton:pressed {
	color: rgb(255, 127, 80);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSqrt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 127, 80);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLg">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 127, 80);
	font-size: 30px;
	border: 0px
}

QPushButton:pressed {
	color: #F7F6F1;
}</string>
    </property>
    <property name="text">
     <string>lg</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionStandart_Mode"/>
    <addaction name="actionCredit_Mode"/>
   </widget>
   <addaction name="menuMode"/>
  </widget>
  <action name="actionStandart_Mode">
   <property name="text">
    <string>Standart Mode</string>
   </property>
  </action>
  <action name="actionCredit_Mode">
   <property name="text">
    <string>Credit Mode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
